package.json

```diff
@@ -73,7 +73,8 @@
     "supertest": "^3.0.0",
     "swagger-stats": "^0.95.6",
     "uuid": "3.1.0",
-    "winston": "^3.0.0"
+    "winston": "^3.0.0",
+    "winston-json-formatter": "false0.9.0"
```

Comment:
Why is this "false0.9.0"?

Reply:
hehe, very good question, I think it something to do with my yarn but I manually remove it.